streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBRangeIteratorTest.java

```diff
@@ -0,0 +1,419 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams.state.internals;
+
+import org.apache.kafka.common.utils.Bytes;
+import org.junit.Test;
+import org.rocksdb.RocksIterator;
+
+import java.util.Collections;
+import java.util.NoSuchElementException;
+
+import static org.easymock.EasyMock.mock;
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.expectLastCall;
+import static org.easymock.EasyMock.replay;
+import static org.easymock.EasyMock.verify;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThrows;
+
+public class RocksDBRangeIteratorTest {
+
+    private final String storeName = "store";
+    private final String key1 = "a";
+    private final String key2 = "b";
+    private final String key3 = "c";
+    private final String key4 = "d";
+
+    private final String value = "value";
+    private final  Bytes key1Bytes = Bytes.wrap(key1.getBytes());
+    private final  Bytes key2Bytes = Bytes.wrap(key2.getBytes());
+    private final  Bytes key3Bytes = Bytes.wrap(key3.getBytes());
+    private final  Bytes key4Bytes = Bytes.wrap(key4.getBytes());
+    private final byte[] valueBytes = value.getBytes();
+
+    @Test
+    public void shouldReturnAllKeysInTheRangeInForwardDirection() {
+
+        final RocksIterator rocksIterator = mock(RocksIterator.class);
+        rocksIterator.seek(key1Bytes.get());
+        expect(rocksIterator.isValid())
+            .andReturn(true)
+            .andReturn(true)
+            .andReturn(true)
+            .andReturn(false);
+        expect(rocksIterator.key())
+            .andReturn(key1Bytes.get())
+            .andReturn(key2Bytes.get())
+            .andReturn(key3Bytes.get());
+        expect(rocksIterator.value()).andReturn(valueBytes).times(3);
+        rocksIterator.next();
+        expectLastCall().times(3);
+        replay(rocksIterator);
+        final RocksDBRangeIterator rocksDBRangeIterator = new RocksDBRangeIterator(
+                storeName,
+            rocksIterator,
+            Collections.emptySet(),
+            key1Bytes,
+            key3Bytes,
+            true
+        );
```

Comment:
```suggestion
        final RocksDBRangeIterator rocksDBRangeIterator = new RocksDBRangeIterator(
            storeName,
            rocksIterator,
            Collections.emptySet(),
            key1Bytes,
            key3Bytes,
            true
        );
```
Please also fix the other wrong indentations.